### DISCUSSION TITLE
How can I remove saxphone sound in a song? Anyone could help?

### DISCUSSION THREAD

**Discussion by b106768** [Anjok07/ultimatevocalremovergui#1215] [Q&A] [answered]
Im a saxphone musician. I need an instrument song removed saxphone sound. Could anyone tell me how to do this?

**Comment by tobocop74** [comment, marked as answer]
Im getting great results with the downloadable VR-model "17_HP_Wind_Inst_UVR". Tested it on a Candy Dulfer song just now and was pretty impressed. :)

**Reply by b106768** [reply]
Thank you buddy. I gotta a try.

**Reply by thereals** [reply]
Where can I download the models? I need one for removing guitar.
